parquet-column/src/test/java/org/apache/parquet/schema/TestTypeBuilders.java

```diff
@@ -205,7 +207,7 @@ public void testFixedLengthEquals() {
     Type f4 = Types.required(FIXED_LEN_BYTE_ARRAY).length(4).named("f4");
     Type f8 = Types.required(FIXED_LEN_BYTE_ARRAY).length(8).named("f8");
     Assert.assertFalse("Types with different lengths should not be equal",
-        f4.equals(f8));
+            f4.equals(f8));
```

Comment:
There are a lot of non-functional changes to the white space that look non-standard.